logger/logger_writer.go

```diff
@@ -66,6 +68,26 @@ type logEntry struct {
 	tee  bool
 }
 
+type logEntryStructured struct {
+	Time    time.Time `json:"time"`
+	Level   string    `json:"level"`
+	Prefix  string    `json:"prefix"`
+	Message string    `json:"msg"`
```

Comment:
Can we change this to `message` to match zerolog?
```suggestion
	Message string    `json:"message"`
```

Example from zerolog:
```json
{
"level":"warn",
"requestID":"insights_162c07ce-b6a6-41fd-a08b-1c3dc29db242",
"companyID":3041,
"userID":6643,
"error":"pq: column \"kt_aws_src_region\" does not exist",
"query":"/* service: insights , insightName: comparison.aws.region_to_region.internal */ SELECT    *,    (f_sum_both_bytes * 8) / (1661863801 - 1661259001) AS bps  FROM    (      SELECT        kt_aws_src_region,        kt_aws_dst_region,        sum(both_bytes) as f_sum_both_bytes,        rank() OVER (          ORDER BY            sum(both_bytes) DESC        ) AS rank      FROM        all_devices      WHERE        i_fast_dataset = TRUE        AND app_protocol IN (0, 1, 2, 3, 4)        AND i_sub_limit = 300        AND simple_trf_prof = 'internal'        AND kt_cloud_provider = $_kntq$AWS$_kntq$        AND ctimestamp > 1661259001        AND ctimestamp <= 1661863801        AND kt_aws_src_region != kt_aws_dst_region        AND i_device_type != 'router'      GROUP BY        kt_aws_src_region,        kt_aws_dst_region      ORDER BY        f_sum_both_bytes DESC      LIMIT 100    ) sq1",
"time":"2022-08-30T12:50:02Z",
"caller":"broker.go:391",
"message":"failed to run query"
}
```

Reply:
Yep, sure. See 05c2e69